### ISSUE TITLE
Switzerland

### ISSUE THREAD

**Issue by betatim** [betatim/build-a-binder#2]
Notes, comments and organisational things about the event in Switzerland.

**Comment by matthewfeickert** [comment]
Depending on the date I can come from CERN to help with the event. I can also help with producing materials.

**Comment by gedankenstuecke** [comment]
It seems like this could make a good fit for the http://rcs18.ethz.ch/?

**Comment by betatim** [comment]
Yes and no. Why not? My hope/aim is to reach people who aren't researchers as well as researchers and RCS is probably full of researchers. The ambitious text in the proposal says we will invite researchers, journalists and educators. (Let's move discussion about how realistic that is into #6.)

As I am invited to speak at RCS and one of the interesting things I do related to research is Binder I think I will talk about Binder there anyway ;)

**Comment by gedankenstuecke** [comment]
Yeah, I thought it might be a good event to do a researcher focused workshop at. And the Open Data CH people might have connections to less researchy audiences? :)

**Comment by heluc** [comment]
Tim, let me know how I can help.

I believe I can: a) find a room at EPFL if this is the location of choice; b) find local helpers; c) advertise on campus; d) use my other networks to advertise to journalists and educators (but also to admin people e.g. from EPFL central services, infrastructure, education data etc.)

**Comment by betatim** [comment]
For me EPFL would be a super choice. So I think taking up this offer would be a good move.

@fthoele mentioned interest as well, does EPFL work for you?

**Comment by fthoele** [comment]
Absolutely!

**Comment by heluc** [comment]
Ok good. I'll start to investigate. Any requirements? (number of people, infrastructures, catering, etc). Also the earlier you come up with a good date, the better.

**Comment by gedankenstuecke** [comment]
For the date: If the date at EPFL would be the week before RCS (or right after) I'd be in the neighborhood to help out 😂

**Comment by gedankenstuecke** [comment]
If I could be super picky I'd go for right before the summer school as I'll be in Europe starting August 29th for MyData (https://mydata2018.org/) and I have some spare time between the two events 😂 

But after the summer school works too :-)

**Comment by gedankenstuecke** [comment]
Yeah, the weekday/weekend debate is hard to solve: 
For people with families and especially kids I've heard both cases argued but I think in this setup it would be easier during work hours (For some people it's easier to organize child care on weekends, but as this is a 'regional' event I guess this is less applicable if it's limited to regular work hours). 
But then I've been on the other side of that debate: for 4 1/2 years I always had to take days off to go to open* conferences, so weekends were easier to me. 
A potential solution could be to make sure the event is advertised with a broad focus, so that people from many walks of live can argue that this is work related?

ad do a small version at your institute/etc: I'm not sure, because this might a) drive lots of people away and b) might only yield people who're already somewhat comfortable with the topic?

**Comment by fthoele** [comment]
Re: number of people: In my experience, ca. 25 is the critical point above which workshops become too anonymous and more like lectures without any connection between/to participants.

**Comment by heluc** [comment]
So, we're left with 8th (Saturday -> weekend) vs 14th (Friday -> weekday) right? Or any other option I should take into consideration? Should we vote? ;) I'm in favour of week day as most of our audience will be researchers (flexible) or specialists (teachers/journalists) who can argue it should be part of their training.. The earlier we agree, the higher the chance I can book a nice room ;) (Tim, what did you think of the one I had for the summer school?)

**Comment by betatim** [comment]
The summer school room was nice!

My vote on Luc's comment (😄) is Friday. Let's use 🎉  as a vote for Saturday. If you disagree with both then comment.

**Comment by heluc** [comment]
Friday then? I'll book the room, then we can discuss the details, ok @betatim?

**Comment by betatim** [comment]
Sounds like a plan to me.

**Comment by betatim** [comment]
Updated top comment with the data as well.

**Comment by heluc** [comment]
Great! Should we build a list of helper/potential participants we should contact in a different issue?

**Comment by RaoOfPhysics** [comment]
@heluc: Add me to said list. I expect to be around and can help.

**Comment by betatim** [comment]
> Great! Should we build a list of helper/potential participants we should contact in a different issue?

Sounds like a good idea. I will add some more labels so we can tag issues for which event they belong to. We could also use a hackmd for more document like things and link them from the issue (has the disadvantage that people don't get notified)

**Comment by betatim** [comment]
@heluc  do you have a suggestion what we do for coffee break and lunch?

My default would be to ask the university to make an offer for catering the coffee breaks and lunch in the canteen for attendees (we then give a voucher to people). We bring soft drinks/water from a supermarket.

If you have a favourite place that does delivery and we can organise plates+cutlery etc that would also work for lunch.

**Comment by heluc** [comment]
@betatim are you planning to open registration for Lausanne soon? I think it's important to give people some notice as it's a busy time of the year. Also, I need some time to arrange catering based on number of people.

**Comment by betatim** [comment]
Yes, let's do that.

https://docs.google.com/forms/d/e/1FAIpQLSfXZYqB2HVSD3Slc45Tw095iFbIcP3FmhCUNeGnyl23c6bZnQ/viewform is the form. When should we start selecting people/close registration? End of August?

* [ ] update webpage with registration link and date

**Comment by RaoOfPhysics** [comment]
@betatim: I think it makes sense to give people a minimum of two-weeks' notice before the event so they can (a) confirm their participation and (b) make arrangements to travel etc. I know it's an expectation that they agree to both when they apply, but in case they need to…

**Comment by heluc** [comment]
@betatim @RaoOfPhysics realistically, we can expect people to come from > 1 hour of EPFL (i.e. Geneva, CERN, Bern, Fribourg, Lausanne) but in any case, we should give a deadline for 15 August, see how many people sign up and extend until the end of the month if necessary. Deal? Better to advertise the event now, before mid-July, so that people are not on holiday and miss the announcement.

**Comment by heluc** [comment]
Also @betatim can you give me the rights to edit the form, or fix a couple of typos? e.g. "At the evenT would YOU like to share examples or facilitate a session?"

**Comment by betatim** [comment]
We will start advertising this on Wednesday 4 July (I think).

**Comment by betatim** [comment]
@heluc time to advertise?

**Comment by heluc** [comment]
@betatim yes! but let's just catch up on slack this afternoon for the last details if possible
